personalpalate/orm/model.py

```diff
@@ -1,46 +1,49 @@
 from datetime import date
 from typing import Optional
-
 from sqlmodel import SQLModel, Field
 from sqlalchemy import PrimaryKeyConstraint
+from enum import Enum
 
+class Category(Enum):
+    dinner = "dinner"
+    lunch = "lunch"
+    breakfast = "breakfast"
 
 class AccountDTO(SQLModel, table=True):
     email: str = Field(primary_key=True)
-    followable: bool
+    followable: bool = Field(default=False)
 
 class Account(AccountDTO):  # Subclass to avoid sending password to client
     password: str
 
 class Follow(SQLModel, table=True):
-    email: str = Field(default=None, foreign_key="account.email")
-    followingEmail: str = Field(default=None, foreign_key="account.email")
+    email: str = Field(foreign_key="accountdto.email")
```

Comment:
These should all be to the account table, accountDTO is just a python object, not a table